js/main.js

```diff
@@ -13,4 +13,14 @@ $(function () {
     var view = new GeneralView(core);
 
     window.atm = core;
+});
+
+$(function () {
```

Comment:
Move this configuration `cardLayoutView.js`, which should be constructor function with configuration.